Apps/WebClient/src/ClientApp/src/components/DependentCardComponent.vue

```diff
@@ -109,10 +123,43 @@ export default class DependentCardComponent extends Vue {
     private immunizations: ImmunizationEvent[] = [];
     private recommendations: Recommendation[] = [];
     private isDataLoaded = false;
+    private isGeneratingReport = false;
     private isImmunizationDataLoaded = false;
+    private reportFormatType = ReportFormatType.PDF;
+    private ReportFormatType: unknown = ReportFormatType;
+    private dependentTab = 0;
 
     private selectedTestRow!: Covid19LaboratoryTestRow;
 
+    private get headerData(): ReportHeader {
+        return {
+            phn: this.dependent.dependentInformation.PHN,
+            dateOfBirth: this.formatDate(
+                this.dependent.dependentInformation.dateOfBirth || ""
+            ),
+            name: this.dependent.dependentInformation
+                ? this.dependent.dependentInformation.firstname +
+                  " " +
+                  this.dependent.dependentInformation.lastname
+                : "",
+            isRedacted: false,
+            datePrinted: new DateWrapper(new DateWrapper().toISO()).format(),
```

Comment:
Is the `.toISO()` needed or can this line be simplified? e.g.:
```
datePrinted: new DateWrapper().format(),
```

Reply:
Changed.